### DISCUSSION TITLE
Where are imported books saved in Windows?

### DISCUSSION THREAD

**Discussion by f4wired** [edrlab/thorium-reader#2051] [Q&A] [answered]
I've imported a book and deleted the original file. The imported book is still accessible through Thorium so I presume the file is being copied and stored elsewhere. Where is it being stored? The program is in `C:\Users\{NAME}\AppData\Local\Programs\EDRLab.ThoriumReader` but i can't seem to find another copy of the book anywhere.

**Comment by f4wired** [comment, marked as answer]
Nevermind, they are under AppData\Roaming\EDRLab.ThoriumReader\publications\

**Comment by gautierchomel** [comment]
There is a _Save a copy of the file_ option available from the Book information panel. See the [Function section of the Thorium Reader Documentation website](https://thorium.edrlab.org/docs/400_ressources/401_functions/).

**Comment by ximosanc** [comment]
Mine was all the way here:
C:\Users\{NAME}\AppData\Local\Packages\EDRLab.ThoriumReader_r3hax6t39xm4t\LocalCache\Roaming\EDRLab.ThoriumReader\publications\

**Reply by danielweck** [reply]
Thank you for reporting.
This looks like the specific filesystem location where Thorium's dedicated config/storage folder is created, when Thorium is installed by the Windows App Store. The executable installer downloaded from GitHub releases causes Thorium to use `C:\USERNAME\AppData\Roaming\EDRLab.ThoriumReader\`. There is nothing we can do to change this Operating System behaviour, I am afraid.

**Comment by gautierchomel** [comment]
Updating the support info as we have now this in a FAQ: https://thorium.edrlab.org/en/th3/400_ressources/430_faq/#localStorage

**Reply by EricBright** [reply]
The mentioned URL seems to be dead. Is there an updated link that can be had?

**Reply by naglis** [reply]
https://thorium.edrlab.org/en/th3/400_resources/430_faq/#localStorage
